test/parallel/test-fs-watch-file-nonexistent.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var fs = require('fs');
+
+var filename = '/path/to/file/that/does/not/exist';
+
+fs.watchFile(filename, function() {
+  throw new Error('callback should not be called for non-existent files');
```

Comment:
@bnoordhuis Yup, if the callback is not fired then the users will not know if the code is working or not. This could be a debugging nightmare.